CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.0.2 (unreleased)
```

Comment:
Looks like we're introducing new functionality here? This should be `v1.1.0` in that case.